core/trino-main/src/main/java/io/trino/server/security/oauth2/OAuth2ServiceModule.java

```diff
@@ -72,7 +72,7 @@ protected void setup(Binder binder)
     @Provides
     @Singleton
     @ForJwk
-    public static URI createJwkAddress(OAuth2Config config)
+    public static URI createJwksAddress(OAuth2Config config)
```

Comment:
push trivial unrelated changes as first commits in PRs. That way you will make it sure that they won't conflict with other changes that you are doing, it will be clear that these are pre-exisiting issues and it will be easier to extract them to separate PR if needed.